scripts/rollup/build-all-release-channels.js

```diff
@@ -2,6 +2,7 @@
 
 /* eslint-disable no-for-of-loops/no-for-of-loops */
 
+const crypto = require('node:crypto');
```

Comment:
fyi, even if I install the latest dependencies i cannot run `yarn build react-dom` without this erroring on `cannot find module`

Reply:
I had a habit of using node 14 b/c we in theory support versions back that far for React. this change requires Node 16 or newer it seems. Not necessarily against that but I may just need to update my default to be v16

Reply:
alternatively you could conditionally require `node:crypto` or `crypto` depending on version and/or if one throws

Reply:
Oh, didn't know we're supporting old node versions in the build. I think just `crypto` works too.